pkg/hubble/parser/threefour/parser.go

```diff
@@ -674,11 +674,11 @@ func decodeNetworkInterface(tn *monitor.TraceNotify, dbg *monitor.DebugCapture)
 	}
 }
 
-func decodeProxyPort(dbg *monitor.DebugCapture) uint32 {
+func decodeProxyPort(dbg *monitor.DebugMsg) uint32 {
 	if dbg != nil {
 		switch dbg.SubType {
-		case monitor.DbgCaptureProxyPre,
-			monitor.DbgCaptureProxyPost:
+		case monitor.DbgProxyPre,
+			monitor.DbgProxyPost:
 			return byteorder.NetworkToHost(dbg.Arg1).(uint32)
 		}
```

Comment:
The function `decodeProxyPort(...)` is called by `parser.Decode()` which uses `monitor.DebugCapture{}`[1] as the argument. But since I've changed the debug message from capture to regular message. Should I remove this piece of code? 

[1]: https://github.com/cilium/cilium/blob/b99403fe3b7a2be121e02b86619b36fb2c85e798/pkg/hubble/parser/threefour/parser.go#L224